host-bin/enter-chroot

```diff
@@ -684,10 +684,20 @@ if [ -n "$NOLOGIN" ]; then
         fi
     fi
 else
-    # Run rc.local
+    # Check and run rc.local
     if [ -n "$firstrun" -a -x "$CHROOT/etc/rc.local" ]; then
+        
```

Comment:
I think this would be more readable (and without spacer lines around it):

```
(sleep 10; echo "Still running chroot's rc.local..." 2>&1) &
noticepid="$!"
```